routing/localchans/manager.go

```diff
@@ -311,14 +311,28 @@ func (r *Manager) updateEdge(tx kvdb.RTx, chanPoint wire.OutPoint,
 	edge *models.ChannelEdgePolicy,
 	newSchema routing.ChannelPolicy) error {
 
+	channel, err := r.FetchChannel(tx, chanPoint)
```

Comment:
Short background why we need this change or something similar:

The first version we tried on a node with a lot of missing edges, we did not set a correct timestamp hence the `edgeInfo` was created but the channelEdgePolicy was rejected because it had the time{} set. Now every time we would fetch the outgoing channels from the graphDB we would always fail because the edge would be `nil`. Moreover this is counterproductive anyways and we should fix the situation where we have still an `unkownPolicy` set. So we decided to always add a new edge policy in case an edge does still not have a policy set.

Reply:
Thanks for that elaboration

Reply:
Think it's a hacky way to fix the issue and may hide an important issue that why the `edge` is nil here.

> Moreover this is counterproductive anyways and we should fix the situation where we have still an unkownPolicy set. So we decided to always add a new edge policy in case an edge does still not have a policy set.

Isn't this is what `createEdge` for, hence the new cli flag `createMissingEdge` is for?